project_report/__init__.py

```diff
@@ -1,499 +0,0 @@
-import os
```

Comment:
Has this content all been removed, or shifted elsewhere?

Reply:
Most of the functionalities have moved to `project_information.py` some were removed completely